addons/maintenance/models/maintenance.py

```diff
@@ -39,7 +39,7 @@ def _compute_fold(self):
     maintenance_ids = fields.One2many('maintenance.request', 'category_id', copy=False)
     maintenance_count = fields.Integer(string="Maintenance", compute='_compute_maintenance_count')
     alias_id = fields.Many2one(
-        'mail.alias', 'Alias', ondelete='cascade', required=True,
+        'mail.alias', 'Alias', ondelete='restrict', required=True,
```

Comment:
when we uninstall mrp_maintenance module , due to ondelete='cascade' and due to this reference in mrp_maintencance  demo record : field name="alias_id" ref="maintenance.mail_alias_equipment" 
it's remove record from maintenance module. 
then when we try to install again mrp_maintenance we got traceback : external id reference not found.. (maintenance.mail_alias_equipment)